app/bundles/CoreBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -25,7 +25,7 @@ public static function getSubscribedEvents()
      */
     public function onBuildJs(BuildJsEvent $event): void
     {
-        $js = <<<'JS'
+        $js = <<<'JS_WRAP'
 // Polyfill for CustomEvent to support IE 9+
 (function () {
```

Comment:
From PHP 7.3, the now/heredoc can be cut by the same keyword despite the spacing.
That's why it's safer to use different values then the string in the contents.

This change makes it safer.

See:
https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#sensitiveherenowdocrector